qemu/tests/cfg/block_hotplug_negative.cfg

```diff
@@ -0,0 +1,69 @@
+# Below case works stablly on QMP monitor, but have no more test with Human
+# monitor, so suggest to use QMP monitor as default qemu monitor run below test
+#
+- block_hotplug_negative:
+    no RHEL.3.9
+    no ide
+    virt_test_type = qemu
+    type = block_hotplug_negative
+    bootindex_image1 = 0
+    images += " stg0"
+    boot_drive_stg0 = no
+    image_name_stg0 = images/storage0
+    image_size_stg0 = 1G
+    remove_image_stg0 = yes
+    force_create_image_stg0 = yes
+    kill_vm_on_error = yes
+
+    variants:
+        - one_pci:
+            blk_num = 1
+
+    variants:
+        - fmt_qcow2:
+            image_extra_params = ""
+            img_format_type = qcow2
+            image_format_stg0 = qcow2
+            image_format_stg1 = qcow2
+        - fmt_raw:
+            image_extra_params = ""
+            img_format_type = raw
+            image_format_stg0 = raw
+            image_format_stg1 = raw
+
+    variants:
+        - block_virtio:
+            pci_type = virtio-blk-pci
+            drive_format_stg0 = virtio
+            drive_format_stg1 = virtio
+            get_disk_pattern = "^/dev/vd[a-z]*$"
+            virtio_blk:
+                no ppc64 ppc64le
+        - block_scsi:
+            pci_type = scsi-hd
+            drive_format_stg0 = scsi-hd
+            drive_format_stg1 = scsi-hd
+            get_disk_pattern = "^/dev/sd[a-z]*$"
+            virtio_blk:
+                need_controller = yes
+                controller_model = virtio-scsi-pci
+
+    variants:
+        - @default:
+            only block_virtio.fmt_raw.one_pci
+            block_virtio:
```

Comment:
for default, does it need add 'x_data_plane=on' ?

Reply:
It can be other params, 'x_data_plane=on' is just a example.